client/components/QuestionsAndAnswers/MasterQA.jsx

```diff
@@ -4,18 +4,33 @@ import MoreQuestions from './MoreQuestions';
 import QuestionList from './QuestionList';
 import SearchQuestion from './SearchQuestion';
 import Data from './dummyData';
+import { getQuestions, getAnswers } from './helpers/helpers';
 import { MasterContainer, Btn, Scroller } from './styles/MasterQA.style';
 
 // const axios = require('axios');
 
 const MasterQA = () => {
-  const sortedQuestions = Data.questions.results.sort(
+  const [questionsFromAPI, setQuestionsFromAPI] = useState([]);
+  const [questionsLength, setQuestionsLength] = useState(2);
+  const sortedQuestions = questionsFromAPI.sort(
```

Comment:
`.sort` mutates it's input ([docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort)). In React, mutating state directly is not good practice.

Reply:
Thanks for catching it, I forgot to add .slice() back before .sort()